WordPress/Classes/ViewRelated/System/Notices/NoticePresenter.swift

```diff
@@ -70,6 +70,14 @@ class NoticePresenter: NSObject {
 
         super.init()
 
+        // Keep the window visible but hide it on the next run loop. If we hide it immediately,
+        // the window is not automatically resized when the device is rotated. This issue
+        // only happens on iPad devices.
```

Comment:
Can we change this comment to mention that the problem appears to be only in the simulator?

That way we can revisit it in the future if this code gets refactored.

Reply:
Agreed 👍